scan/lib/scan/options.rb

```diff
@@ -70,19 +70,30 @@ def self.available_options
                                      type: Boolean,
                                      optional: true),
 
-        # reset simulator
+        FastlaneCore::ConfigItem.new(key: :force_quit_simulator,
+                                     env_name: 'SCAN_FORCE_QUIT_SIMULATOR',
+                                     description: "Enabling this option will automatically killall Simulator processes before the run",
+                                     default_value: false,
+                                     type: Boolean),
         FastlaneCore::ConfigItem.new(key: :reset_simulator,
                                      env_name: 'SCAN_RESET_SIMULATOR',
                                      description: "Enabling this option will automatically erase the simulator before running the application",
                                      default_value: false,
-                                     type: Boolean),
-
-        # reinstall app
```

Comment:
this structuring comment is still valid, right? please keep this.

Reply:
I think it is a group of options (rather than simply restating the name of a single option).
I added a ` simulator management` structural comment for the related three options.

Reply:
Yes, before there were two different "group" comments, one just repeating the obvious as I couldn't find a better label. What you did now think is fine and matches the intention behind those options just fine.